primo/data_parser/__init__.py

```diff
@@ -10,3 +10,5 @@
 # reproduce, distribute copies to the public, prepare derivative works, and
 # perform publicly and display publicly, and to permit others to do so.
 #################################################################################
+
+from .metric_data import ImpactMetrics, EfficiencyMetrics
```

Comment:
Real nitpicky, but could you please arrange the imports alphabetically? (EfficiencyMetrics before ImpactMetrics?)